modules/pushsnap_scriptworker/manifests/settings.pp

```diff
@@ -72,7 +71,6 @@
 
     $script_config_content      = {
         work_dir                 => $work_dir,
-        schema_file              => "${root}/lib/python${python3_virtualenv_version}/site-packages/pushsnapscript/data/push_snap_task_schema.json",
```

Comment:
Blocker: I think you know it already since you opened https://github.com/mozilla-releng/pushsnapscript/pull/16 => we need to release a new version of pushsnapscript

Reply:
Fortunately, `pushsnapscript` already had a default `schema_file` set. [This PR](https://github.com/mozilla-releng/pushsnapscript/pull/16) removed `schema_file` from the example config, but didn't change any underlying logic :smile: